react/src/SearchVariants/components/Autocomplete.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { Autocomplete } from "@material-ui/lab";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import { useGenesQuery } from "./../../hooks/genes";
+import { OutlinedInput, useTheme } from "@material-ui/core";
+import { Search } from "@material-ui/icons";
```

Comment:
Import `node_modules/` dependencies first in alphabetical order, then our own code. If importing from mui core, use the destructured import. For the first-party import, `./` isn't needed since it's in a parent directory.

Reply:
updated in [5195119](https://github.com/ccmbioinfo/stager/commit/51951199963d68492f614fc067aa268ed2dca411)